packages/core/strapi/lib/types/__tests__/test.utils.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const path = require('path');
+const { fromFile } = require('ts-zen');
+
+const DEFINITIONS_ROOT = path.join(__dirname, 'definitions');
+
+/**
+ *
+ * @param {string} filePath
+ *
+ * @return {import('ts-zen').AssertTypeSelector}
+ */
+const createTypeSelector = (filePath) => {
+  return fromFile(path.join(DEFINITIONS_ROOT, filePath), {
+    compilerOptions: { strict: true },
+    ignoreProjectOptions: true,
```

Comment:
Let's add a TODO since we'll need to remove those options when strapi/strapi will be converted to TS